seed_db.sh

```diff
@@ -131,93 +131,93 @@ else
+        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 6, 'Lee’s hand shook as he filled the delicate cups. 
             He drank his down in one gulp. “Don’t you see?” he cried. ');"
         
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 8, '“The American Standard translation orders men to triumph over sin, 
+        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 7, '“The American Standard translation orders men to triumph over sin, 
             and you can call sin ignorance. ');"
         
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 9, 'The King James translation makes a promise in ‘Thou shalt,’ 
+        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 8, 'The King James translation makes a promise in ‘Thou shalt,’ 
             meaning that men will surely triumph over sin. ');"
         
-        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 10, 'But the Hebrew word, the word timshel—‘Thou mayest’—that gives a choice. 
+        docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 9, 'But the Hebrew word, the word timshel—‘Thou mayest’—that gives a choice. 
             It might be the most important word in the world. 
             That says the way is open. That throws it right back on a man.
             For if ‘Thou mayest’—it is also true that ‘Thou mayest not.’ Don’t you see?”');"
     done 
 
     for STORY_TRANSLATION_ID in 1 4 6 8 9 10  
     do 
-        for i in 1 2 3 4 5 6 7 8 9 10
+        for i in 0 1 2 3 4 5 6 7 8 9
         do
             docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', $i, '');"
         done 
     done 
 
     for STORY_TRANSLATION_ID in 12 
     do 
-        for i in 1 2 3 4 5 6 7 8 9 
+        for i in 0 1 2 3 4 5 6 7 8 9
```

Comment:
This leaves a total of 11 lines (10 blank and 1 filled). Let's try to keep the amount of lines consistent to 10 in total. 

`for i in 0 1 2 3 4 5 6 7 8` should be good and also remember to change the line number for the snippet below to 9.

Reply:
story_translation id=12 corresponds to story id = 2. 
`SELECT * FROM story_contents WHERE story_id=5;` results in 10 total lines, indexed 0-9
`SELECT * FROM story_translation_contents WHERE story_translation_id=12;` results in 11 total lines, indexed 0-10 where every line is empty except for line index 10.
@e-wai I think this forloop should run 0-8 due to line 225